src/matchers/toMatchText/index.test.ts

```diff
@@ -0,0 +1,82 @@
+import toMatchText from "."
+import { assertSnapshot } from "../tests/utils"
+
+expect.extend({ toMatchText })
+
+describe("toMatchText", () => {
+  afterEach(async () => {
+    await page.setContent("")
+  })
+  describe("selector", () => {
+    it("positive frame", async () => {
+      await page.setContent(`<iframe src="https://example.com"></iframe>`)
+      const iframe = await page.$("iframe")
+      await expect(iframe!).toMatchText("h1", /.*ample Domai.*/)
+    })
+    it("positive", async () => {
+      await page.setContent(`<div id="foobar">Bar</div>`)
+      await expect(page).toMatchText("#foobar", /[B|b]ar/)
+    })
+    it("negative", async () => {
+      await page.setContent(`<div id="foobar">zzzBarzzz</div>`)
+      await assertSnapshot(() => expect(page).toMatchText("#foobar", /[b]ar/))
+    })
+    describe("with 'not' usage", () => {
+      it("positive in frame", async () => {
+        await page.setContent(`<iframe src="https://example.com"></iframe>`)
+        const iframe = await page.$("iframe")
+        await expect(iframe!).not.toMatchText("h1", /ab+c/)
+      })
+      it("positive", async () => {
+        await page.setContent(`<div id="foobar">Bar</div>`)
+        await expect(page).not.toMatchText("#foobar", /exam.*[1|2]/)
+      })
+      it("negative", async () => {
+        await page.setContent(`<div id="foobar">zzzBarzzz</div>`)
+        await assertSnapshot(() =>
+          expect(page).toMatchText("#foobar", /not-existing/)
```

Comment:
This snapshot should be asserting `.not.toMatchText`.

```suggestion
          expect(page).not.toMatchText("#foobar", /z+Bar/)
```